### PR TITLE
Added Configuration files and properties

### PR DESCRIPTION
**Pull request by JacketPants12**
- Added Properties file
- Added Firebase configuration file
- Delete FirebaseDatastore
- Refactored collection name variable

### CODE REVIEW

**Review by youssefsoli — CHANGES_REQUESTED** [review]
Not a full review, just a single comment for now

File: src/main/java/ca/utoronto/fitbook/adapter/persistence/firebase/FirebaseConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        }
+    }
+
+    @ResponseStatus(value= HttpStatus.INTERNAL_SERVER_ERROR, reason="Failed to configure to Firebase")
+    public static class FirebaseConfigurationException extends RuntimeException {
+        public FirebaseConfigurationException() {
+            super("Failed to connect to Firebase");
+        }
+    }
+}
```
Comment (youssefsoli) [inline src/main/java/ca/utoronto/fitbook/adapter/persistence/firebase/FirebaseConfiguration.java:56]: Add some javadocs

**Review by youssefsoli — CHANGES_REQUESTED** [review]

**Review by Ludosss — APPROVED** [review]
looks good

**Review by youssefsoli — APPROVED** [review]
Looks good now

**Review by jpablo2002 — APPROVED** [review]
lgtm

**Review by Ludosss — APPROVED** [review]

**Pull request merged**